src/components/__tests__/CurrentRefinements.js

```diff
@@ -5,20 +5,48 @@ import { __setState } from '../../mixins/widget';
 jest.mock('../../mixins/widget');
 jest.mock('../../mixins/panel');
 
-it('accepts a transformItems prop', () => {
+it('TransformItems happens after excludedAttributes', () => {
   __setState({
-    refinements: [],
+    refinements: [
+      { type: 'query', query: 'hi there everyone!' },
+      { type: 'disjunctive', attribute: 'brand', value: 'Insignia™' },
+    ],
+  });
+
+  const transformItems = items =>
+    items.filter(({ attribute }) => attribute === 'query');
```

Comment:
This doesn't seem to test if `transformItems` is applied after `excludedAttributes`.
A way to effectively test it would be to overwrite the whole content with `transformItems`.